README.md

```diff
@@ -63,12 +63,20 @@ paymentOptions.setPaymentType(PaymentType.ALL);
 paymentOptions.setStoredCardList(paymentMethodItems);
 
 //set merchant customizations such as page title, icon, and footer text (html)
-paymentOptions.setMerchantTitle("Lucy's Shop");
```

Comment:
these 2 methods are not available

Reply:
Correct, instead we should tell people to use setHeader() and setFooter() these take a resource id, and will be deflated when the UI is displayed.